<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="pixmapWidget">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="newFileAction"/>
    <addaction name="saveAction"/>
    <addaction name="loadAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Сцена</string>
    </property>
    <addaction name="addPolygonAction"/>
    <addaction name="addLightAction"/>
    <addaction name="editSceneAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Отрисовка</string>
    </property>
    <addaction name="renderAction"/>
    <addaction name="separator"/>
    <addaction name="simpleRendAction"/>
    <addaction name="parallelRendAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFileAction">
   <property name="text">
    <string>Новый файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Записать...</string>
   </property>
   <property name="toolTip">
    <string>Записать...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="loadAction">
   <property name="text">
    <string>Загрузить...</string>
   </property>
   <property name="toolTip">
    <string>Загрузить...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="renderAction">
   <property name="text">
    <string>Начать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="addPolygonAction">
   <property name="text">
    <string>Добавить многоугольник...</string>
   </property>
   <property name="toolTip">
    <string>Добавить многоугольник...</string>
   </property>
  </action>
  <action name="addLightAction">
   <property name="text">
    <string>Добавить источник света...</string>
   </property>
   <property name="toolTip">
    <string>Добавить источник света...</string>
   </property>
  </action>
  <action name="editSceneAction">
   <property name="text">
    <string>Преобразовать сцену...</string>
   </property>
   <property name="toolTip">
    <string>Преобразовать сцену...</string>
   </property>
  </action>
  <action name="simpleRendAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Обычная</string>
   </property>
  </action>
  <action name="parallelRendAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Параллельная</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
